### PR TITLE
Add an embed/widget API to externally control embed iframe

### PR DESCRIPTION
**Pull request by brycedev**
I've modified the build process to include the compiled JS file in build/ . This file will need to be setup to be served somewhere of your choosing, and can then be accessed by developers. 

I have a [branch of the React Player library](https://github.com/brycedev/react-player/tree/features/audius-integration) that has a working example of an integration. 

To test: 
Spin up the local Bedtime server on this branch. 
Pull my React Player branch and serve the project :
Point lines 292 & 293 of src/demo/App.js to your local embed server address. 
Point line 15 of src/patterns.js to your local embed server address.
Point line 6 of src/players/Audius.js to locally served build/api.js file  

I hope this is both clear and useful. Please let me know if I can help test/integrate this.

### CODE REVIEW

**Comment by raymondjacobson** [comment]
😍😍
Per offline discussion, approach looks great! Just want to not introduce coffee as a dep. Tag me when you want a review & we'll get this deployed ASAP

**Comment by brycedev** [comment]
@raymondjacobson PR is ready for review! Cheers

File: src/components/track/TrackPlayerContainer.jsx [hunk 1]
```diff
@@ -64,6 +66,29 @@ const TrackPlayerContainer = ({
...
+          const messageData = JSON.parse(e.data)
+          const { from, method, value } = messageData
+          if(from && from == 'audiusapi'){
+            if(method === 'togglePlay') didTogglePlay()
+            if(method === 'stop') stop()
+            if(method === 'seekTo') seekTo(value)
+            if(method === 'getPosition') return position
+          }
+        } catch (error) {
+
```
Comment (raymondjacobson) [inline src/components/track/TrackPlayerContainer.jsx:82]: maybe just `console.error(error)` this guy to help in debugging if this happens

**Review by raymondjacobson — APPROVED** [review]
Thank you so much! There are some minor lint things, but this codebase has some lint issues as is, so we can go ahead & merge and we clean it up internally :) I played around with it on react-player. Fun!

Really impressed you got this working off of 0 documentation. We can have this deployed first thing monday if that's good with you!

**Comment by raymondjacobson** [comment]
lgtm! let's squash+merge & can deploy immediately :)

**Pull request merged**